package.json

```diff
@@ -1,17 +1,66 @@
 {
   "name": "roboconf-web-administration",
   "version": "0.0.0",
+  "homepage": "http://roboconf.net",
   "description": "Web administration for Roboconf",
-  "main": "index.js",
+  "keywords": [
+    "cloud",
+    "deployment",
+    "orchestration"
+  ],
   "dependencies": {
     "express": "~3.4.7"
   },
   "devDependencies": {
-    "bower": "~1.3.8"
+    "bower": "~1.3.8",
```

Comment:
You should put `bower` in `dependencies` because you use it in the post install script so if you use `npm install -p` to install in production mode the post script install will fail because in production mode only `dependencies` are downloaded.